packages/one-app-bundler/__tests__/webpack/module/webpack.client.spec.js

```diff
@@ -13,12 +13,13 @@
  */
 
 /* eslint-disable global-require */
+const { sync } = require('read-pkg-up');
 const { validateWebpackConfig } = require('../../../test-utils');
 const getConfigOptions = require('../../../utils/getConfigOptions');
 const configGenerator = require('../../../webpack/module/webpack.client');
 
 jest.mock('../../../utils/getConfigOptions', () => jest.fn(() => ({ purgecss: {} })));
-
+jest.spyOn(process, 'cwd').mockImplementation(() => __dirname.split('/__tests__')[0]);
```

Comment:
Is this line meant to be duplicated?
Will the slash prevent this test from working on windows?

Reply:
> Is this line meant to be duplicated?
No, it was a mistake. Fixing it.

Reply:
Updating to `path.sep`.